roles/vault/tasks/bootstrap/main.yml

```diff
@@ -43,7 +43,7 @@
 - include_tasks: ../shared/gen_ca.yml
   vars:
     gen_ca_cert_dir: "{{ vault_pki_mounts.vault.cert_dir }}"
-    gen_ca_mount_path: "{{ vault_pki_mounts.vault.name }}"
+    gen_ca_mount_path: "/{{ vault_pki_mounts.vault.name }}"
```

Comment:
is this redundant with https://github.com/kubernetes-incubator/kubespray/pull/2733/files#diff-ea8ef3015d8cfa7c6d0a3704c1f763d2R3 ?

Reply:
good catch. I will clean that up

Reply:
It's a big refactor to reconsolidate mount path specified outside of the pki mount dict and a separate var... I'll keep it as is for now